src/workerd/api/node/tests/timers-nodejs-test.js

```diff
@@ -0,0 +1,26 @@
+import timers from 'node:timers';
+import { throws } from 'node:assert';
+
+// Tests are taken from:
+// https://github.com/nodejs/node/blob/3fe8027cbb197953bd524115295d7dd13b9ee1fe/test/parallel/test-timers-enroll-invalid-msecs.js
+export const enrollInvalidMsecs = {
+  async test() {
+    [{}, [], 'foo', () => {}, Symbol('foo')].forEach((val) => {
+      throws(() => timers.enroll({}, val), {
```

Comment:
This would not work with the change I suggested above...

But do we really need to test enroll which is unsupported anyway?